dbms/src/Common/filesystemHelpers.cpp

```diff
@@ -0,0 +1,27 @@
+#include <Common/filesystemHelpers.h>
+#include <common/config_common.h>
+#include <Poco/File.h>
+#include <Poco/Path.h>
+
+namespace DB
+{
+
+bool checkFreeSpace(const std::string & path, size_t data_size)
```

Comment:
Not very good name. Can't understand that it's connected with files.
Maybe `hasEnoughSpaceInDirectory(path, required_bytes)`?

Reply:
Maybe add namespace?

Reply:
The fucntion's been renamed. I've thought about a namespace. I think it's better to make it for removing explicit Poco::File usage and then append these functions into it.